src/app/modules/querybuilder/components/querybuilder-editor/display/display-criterion/display-criterion.component.html

```diff
@@ -1,6 +1,9 @@
 <div class="crit-box" fxLayout="row">
   <div fxFlex="15" class="code-part">
     <!--<div *ngFor="let termcode of criterion.termCodes">{{ termcode?.code }}</div>-->
+    <div *ngIf="isinvalid" class="invalid-Criterion">
+      <img src="assets/img/alert.png" alt="Validation Alert" />
```

Comment:
@thkoehler11  alert.png seems to be missing from assets - also note that it might be better to use an icon if possible